### PR TITLE
Adding a null check to the UUIDConverter.java

### PR DESCRIPTION
**Pull request by Baaasty**

### CODE REVIEW

File: sadu-core/src/test/java/de/chojo/sadu/conversion/UUIDConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        // Check if you can still convert with bytes
+        UUID uuid = UUID.fromString("7ccf6e1c-68fc-442d-88d0-341b315a29cd");
+        byte[] uuidAsByte = UUIDConverter.convert(uuid);
+
+        assertEquals(UUIDConverter.convert(uuidAsByte), uuid);
+
+        // Check if the new null check works
+        assertNull(UUIDConverter.convert((byte[]) null));
+    }
+}
```
Comment (yannicklamprecht) [inline sadu-core/src/test/java/de/chojo/sadu/conversion/UUIDConverterTest.java:29]: Missing newline.

File: sadu-core/src/main/java/de/chojo/sadu/conversion/UUIDConverter.java [hunk 2]
```diff
@@ -37,6 +37,8 @@ public static byte[] convert(UUID uuid) {
      * @return new uuid instance
      */
     public static UUID convert(byte[] bytes) {
+        if (bytes == null) return null;
+
```
Comment (yannicklamprecht) [inline sadu-core/src/main/java/de/chojo/sadu/conversion/UUIDConverter.java:41]: Remove empty line.

**Review by yannicklamprecht — CHANGES_REQUESTED** [review]

File: sadu-core/src/main/java/de/chojo/sadu/conversion/UUIDConverter.java (hunk 2 above)
Comment (Baaasty) [reply sadu-core/src/main/java/de/chojo/sadu/conversion/UUIDConverter.java]: ✅

File: sadu-core/src/test/java/de/chojo/sadu/conversion/UUIDConverterTest.java (hunk 1 above)
Comment (Baaasty) [reply sadu-core/src/test/java/de/chojo/sadu/conversion/UUIDConverterTest.java]: ✅

**Pull request merged**